src/main/java/com/coniverse/dangjang/domain/guide/common/service/DayGuideService.java

```diff
@@ -0,0 +1,90 @@
+import com.coniverse.dangjang.domain.guide.common.exception.GuideNotFoundException;
+import com.coniverse.dangjang.domain.guide.exercise.document.ExerciseCalorie;
+import com.coniverse.dangjang.domain.guide.exercise.document.ExerciseGuide;
+import com.coniverse.dangjang.domain.guide.exercise.dto.ExerciseDayGuide;
+import com.coniverse.dangjang.domain.guide.exercise.service.ExerciseGuideSearchService;
+import com.coniverse.dangjang.domain.guide.weight.document.WeightGuide;
+import com.coniverse.dangjang.domain.guide.weight.dto.WeightDayGuide;
+import com.coniverse.dangjang.domain.guide.weight.service.WeightGuideSearchService;
+import com.coniverse.dangjang.domain.user.service.UserSearchService;
+
+import lombok.RequiredArgsConstructor;
+
+/**
+ * 하루 가이드 조회 Service
+ *
+ * @author EVE
+ * @since 1.0.0
+ */
+@Service
+@RequiredArgsConstructor
+public class DayGuideService {
+	private final UserSearchService userSearchService;
+	private final BloodSugarGuideSearchService bloodSugarGuideSearchService;
+	private final WeightGuideSearchService weightGuideSearchService;
+	private final ExerciseGuideSearchService exerciseGuideSearchService;
+
+	/**
+	 * 하루 가이드를 조회한다.
+	 * <p>
+	 * 홈에 필요한 유저 닉네임,날짜,혈당 가이드,체중 가이드,운동 가이드,알림 여부를 조회해 전달한다.
+	 *
+	 * @param oauthId 유저아이디
+	 * @param date    조회 날짜
+	 * @author EVE
+	 * @since 1.0.0
+	 */
+	public DayGuideResponse getDayGuide(String oauthId, String date) {
+		LocalDate localDate = LocalDate.parse(date);
+		String userNickname = userSearchService.findUserByOauthId(oauthId).getNickname();
+		List<TodayGuide> bloodSugarTodayGuide = getBloodSugarTodayGuide(oauthId, localDate);
+		WeightDayGuide weightDayGuide = getWeightGuide(oauthId, date);
+		ExerciseDayGuide exerciseDayGuide = getExerciseGuide(oauthId, localDate);
+		return new DayGuideResponse(userNickname, localDate, bloodSugarTodayGuide, weightDayGuide, exerciseDayGuide, false);
+	}
+
+	/**
+	 * 혈당 하루 가이드를 조회한다.
+	 *
+	 * @param oauthId 유저아이디
+	 * @param date    조회 날짜
+	 * @since 1.0.0
+	 */
+	public List<TodayGuide> getBloodSugarTodayGuide(String oauthId, LocalDate date) {
+		try {
+			return bloodSugarGuideSearchService.findByUserIdAndCreatedAt(oauthId, date).getTodayGuides();
+		} catch (GuideNotFoundException e) {
+			return new ArrayList<>();
+		}
+	}
+
+	public WeightDayGuide getWeightGuide(String oauthId, String date) {
+		try {
+			WeightGuide weightGuide = weightGuideSearchService.findByUserIdAndCreatedAt(oauthId, date);
+			return new WeightDayGuide(weightGuide.getUnit(), weightGuide.getBmi(), weightGuide.getContent());
+		} catch (GuideNotFoundException e) {
+			return new WeightDayGuide("", 0, "");
+		}
+	}
+
+	public ExerciseDayGuide getExerciseGuide(String oauthId, LocalDate date) {
+		try {
+			ExerciseGuide exerciseGuide = exerciseGuideSearchService.findByOauthIdAndCreatedAt(oauthId, date);
+			int sumCalorie = exerciseGuide.getExerciseCalories().stream().mapToInt(ExerciseCalorie::calorie).sum();
```

Comment:
```suggestion
			int sumCalorie = exerciseGuide.getExerciseCalories()
			    .stream()
			    .mapToInt(ExerciseCalorie::calorie)
			    .sum();
```

P3) stream은 한 라인당 `.메서드()`를 하나씩 두는 것이 가독성면에서 더 좋을 것 같아요!